spanner/spanner_snippets/snippet.go

```diff
@@ -1894,7 +1918,351 @@ func createClientWithQueryOptions(ctx context.Context, w io.Writer, database str
 
 // [END spanner_create_client_with_query_options]
 
-func createClients(ctx context.Context, db string) (*database.DatabaseAdminClient, *spanner.Client) {
+// [START spanner_create_backup]
+
+func createBackup(ctx context.Context, w io.Writer, adminClient *spadmindb.DatabaseAdminClient, database, backupID string) error {
+	expireTime := time.Now().AddDate(0, 0, 14)
+	// Create a backup.
+	op, err := adminClient.StartBackupOperation(ctx, backupID, database, expireTime)
+	if err != nil {
+		return err
+	}
+	// Wait for backup operation to complete.
+	backup, err := op.Wait(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Get the name, create time and backup size.
+	fmt.Fprintf(w, "Backup %s of size %d bytes was created at %s\n", backup.Name,
+		backup.SizeBytes, time.Unix(backup.CreateTime.Seconds, int64(backup.CreateTime.Nanos)).Format(time.RFC3339))
```

Comment:
Please declare a separate variable for the `time.Unix` result and don't do a line break in the middle of the function call.

https://github.com/golang/go/wiki/CodeReviewComments#line-length (the "in the middle of function calls" part applies here).